pipeline/splicing/calcVarPriors.py

```diff
@@ -369,11 +381,29 @@ def varInSpliceDonor(variant):
             return True
     return False
 
+def getVarSpliceDonorBounds(variant):
+    '''
+    Given a variant, checks if variant is in a splice donor region
+    If variant is in a splice donor, returns a dictionary with donor region boundaries where variant is located
+    '''
+    inSpliceDonor = varInSpliceDonor(variant)
+    if inSpliceDonor == True:
```

Comment:
a simpler way to write this is:

`if varInSpliceDonor(variant):`

(no need for inSpliceDonor variable)